content/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-3.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Fast data processing pipeline for predicting flight delays using Apache
+  APIs: Kafka, Spark Streaming and Machine Learning (part 3)"
+date: 2021-10-11T21:12:59.805Z
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapR
+---
+```
+"authorDisplayName": "Carol McDonald",
+"tags": "use-cases",
+"publish": "2018-02-08T12:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/data-fabric.html>
+
+**\*Editor's Note:** This is a 3-Part Series, see the previously published posts below:*
+
+* [Part 1 - Spark Machine Learning](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-/)
+* [Part 2 - Kafka and Spark Streaming](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-2/)
+
+Fast Data Processing Pipeline for Predicting Flight Delays Using Apache APIs: Kafka, Spark Machine Learning, Drill, with MapR Event Store and MapR Database JSON (Part 3)
+
+According to Bob Renner, CEO of [Liaison Technologies](https://www.liaison.com/), the possibility to blend machine learning with real-time transactional data flowing through a single platform is opening a world of new possibilities, such as enabling organizations to take advantage of opportunities as they arise. According to [Gartner](https://www.gartner.com/newsroom/id/3812063) Over the next few years, virtually every app, application and service will incorporate some level of machine learning. Leveraging these opportunities requires fast and scalable data processing pipelines.
+
+![MapR Platform](/img/mapr-platform.png)
+
+This is the third in a series of blogs, which discusses the architecture of a data pipeline that combines streaming data with machine learning and fast storage. [The first post](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-/) discussed creating a machine learning model to predict flight delays. The [second post](https://developer.hpe.com/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-2/) discussed using the saved model with streaming data to do real-time analysis of flight delays. This third post will discuss fast storage and analysis with MapR Database, Apache Spark, Apache Drill and OJAI.
+
+**Machine Learning Logistics and Data Pipelines**
```

Comment:
The subheaders in this part of the blog series look much better in the smaller font than the part 2 post. This is good. If you could make the part 2 post have subheaders like this, it would be better in that post.